vignettes/carbon_tracking.Rmd

```diff
@@ -19,6 +19,7 @@ This vignette will walk through Hector's carbon tracking feature and show how to
 Hector's carbon tracking feature helps to trace the movement of carbon between various pools as the model runs. At the user-chosen start date, each pool is
 deemed to be composed of 100% carbon from itself, i.e. at that moment in time 
 the deep ocean pool is 100% composed of carbon from the deep pool. 
+
```

Comment:
@leeyap not sure why I cannot comment directly on this line but at https://github.com/JGCRI/hector/pull/488/files#R5 the ymal header is for a regular markdown not a vignette. 

it should read something like 

```
output: rmarkdown::html_vignette
vignette: >
  %\VignetteEncoding{UTF-8}
  %\VignetteIndexEntry{Example: Carbon tracking with Hector}
  %\VignetteEngine{knitr::rmarkdown}
editor_options: 
  chunk_output_type: console
```